sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln

```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.26228.4
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.ServiceBus.Tests", "tests\Microsoft.Azure.Management.ServiceBus.Tests.csproj", "{90FE65CB-D848-4ED8-97F5-7C967102CEAF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.ServiceBus.Tests", "tests\Management.ServiceBus.Tests.csproj", "{90FE65CB-D848-4ED8-97F5-7C967102CEAF}"
 EndProject
```

Comment:
Suggest to rename csproj files with prefix **Microsoft.Azure**

Reply:
Hi Erich, judging by the failures in the build checks, I'm guessing that the build pipeline is probably looking for the csproj files by name, and that changing them is a breaking change. How should we proceed?

Reply:
After renaming csproj files, you should update **Management.ServiceBus.csproj** to **Microsoft.Azure.Management.ServiceBus.csproj** in test csproj file as well.

Reply:
Do you mean the project reference of the unit tests to the main project? I have done that in the 2nd commit. Let me know if you meant something else.

Reply:
Ok, having actually pushed the changes properly, everything is passing. Sorry about that.